src/SignalR/clients/cpp/src/signalrclient/url_builder.cpp

```diff
@@ -80,7 +80,7 @@ namespace signalr
             const utility::string_t& connection_data, const utility::string_t& query_string,
             const utility::string_t& last_message_id = _XPLATSTR(""), const utility::string_t& groups_token = _XPLATSTR(""))
         {
-            _ASSERTE(command == _XPLATSTR("reconnect") || (last_message_id.length() == 0 && groups_token.length() == 0));
```

Comment:
There are still a bunch of commented-out reconnect tests in url_builder_tests.cpp. Can we remove the entire url_builder_tests.cpp file? Or at least the reconnect test portions of it?